src/module/user/sheet.ts

```diff
@@ -1,14 +1,62 @@
+import { htmlQueryAll } from "@util";
 import type { UserPF2e } from "./document.ts";
 
 /** Player-specific settings, stored as flags on each User */
 export class UserConfigPF2e<TUser extends UserPF2e> extends foundry.applications.sheets.UserConfig<TUser> {
-    override async getData(options: DocumentSheetOptions): Promise<UserConfigData<TUser>> {
-        const data = await super.getData(options);
-        data.actors = data.actors.filter((a) => a.type !== "party");
-        return data;
+    static override PARTS = {
+        tabs: {
+            template: "templates/generic/tab-navigation.hbs",
+        },
+        // Add a new main part, which embeds the original form part
+        main: {
+            template: "systems/pf2e/templates/user/sheet.hbs",
+        },
+
+        ...fu.deepClone(super.PARTS),
+    };
+
+    override tabGroups = {
+        primary: "core",
+    };
+
+    #getTabs() {
+        const DEFAULTS = { group: "primary" as const, active: false, cssClass: "" };
+        const tabs = [
+            { ...DEFAULTS, id: "core", icon: "fa-solid fa-cog", label: "Core" },
+            { ...DEFAULTS, id: "pf2e", icon: "fa-solid fa-cog", label: "PF2E.Pathfinder" },
```

Comment:
I don't know if `Pathfinder` is the best label for this. `Pathfinder Second Edition` like in the settings is probably too long. Maybe `System`?

Reply:
Sure. If you got any idea for what would be good font awesome choices as well I'm all ears.

Reply:
`fa-user` for the first tab and maybe `fa-dice` (like the systems tab in the setup screen) for the other?

Reply:
Done! Good choice.